### PR TITLE
Header/footer/career details/company section/home page subnav

### PR DESCRIPTION
**Pull request by zainaboyedeji**
This is a compilation of all my part of the project. i.e Header,Footer,Company-A section,Career-details,Homepage sub navbar,Company subnavbar.

Career details 1
career details 2
company section 2
company video modal
company-section1
company-subnavbar
company-subnavbar2
cta-modal
cta-modal-sent
Footer
Header
Homepage-subnavbar

### CODE REVIEW

File: scss/pages/_home.scss [hunk 1]
```diff
@@ -1,115 +0,0 @@
-.ctamodal{
```
Comment (SEUNAGBEYE) [inline scss/pages/_home.scss:1]: Where did you move this deleted CSS to?

File: scss/pages/_about.scss [hunk 2]
```diff
@@ -1,3 +0,0 @@
-.companysection2{
```
Comment (SEUNAGBEYE) [inline scss/pages/_about.scss:1]: Did you move this CSS somewhere?

File: scss/abstracts/mixins.scss [hunk 3]
```diff
@@ -1,14 +0,0 @@
-@mixin conTain {
```
Comment (SEUNAGBEYE) [inline scss/abstracts/mixins.scss:1]: Where did you move this to

**Review by SEUNAGBEYE — CHANGES_REQUESTED** [review]

File: scss/pages/_home.scss (hunk 1 above)
Comment (zainaboyedeji) [reply scss/pages/_home.scss:1]: inside the homepage.scss

File: scss/pages/_about.scss (hunk 2 above)
Comment (zainaboyedeji) [reply scss/pages/_about.scss:1]: yes inside company.scss

File: scss/abstracts/mixins.scss (hunk 3 above)
Comment (zainaboyedeji) [reply scss/abstracts/mixins.scss:1]: it is inside reset.scss as restriction

**Review by SEUNAGBEYE — APPROVED** [review]
LGTM

**Comment by SEUNAGBEYE** [comment]
Raise a another PR for the fixes I mentioned on our call

**Pull request merged**